js/index.js

```diff
@@ -26,4 +28,54 @@ for (let i=0; i < skills.length; i++) {
     skill.innerText = skills[i];
 
     skillsList.appendChild(skill);
-}
\ No newline at end of file
+}
+
+const name = document.querySelector("#name");
+const email = document.querySelector("#email");
+const message = document.querySelector("#message");
+
+
+const messageForm = document.querySelector('#form')
+
+
+messageForm.addEventListener('submit', (event) => {
+
+    event.preventDefault();
+
+    // const nameInput = event.target.name
```

Comment:
You are actually really close here. So `event.target.name` is the same object as the `name` variable you defined in line 33. So the assignment was asking you get the `.value` of that and then `console.log` it.